### ISSUE TITLE
Script never fires after download

### ISSUE THREAD

**Issue by tomlarose** [rakshasa/rtorrent#278]
I cannot for the life of me get any script to run after a download is complete.

```shell
method.set_key = event.download.finished,test_123,"execute2=/home/user/test.sh"
```

Here are the permissions for test.sh:

`
-rwxrwxrwx 1 user user   46 Mar  9 21:31 test.sh
`

test.sh is simply:

```shell
#!/bin/bash

date >> /home/user/test.log
```

I'm using Ubuntu 14.04 LTS and rtorrent 0.9.4

What am I doing wrong?  Any help is appreciated!

**Comment by barchan** [comment]
same issue, please help!

**Comment by barchan** [comment]
problem solved (rtorrent 0.9.4)

'method.set_key = event.download.finished, my_script, "execute = /home/user/script.sh, $d.name=, $d.base_path="'

this code work for me.

p.s: don`t forget set chmod +x script.sh
